sap-service-operator/src/main/java/com/sap/cloud/environment/servicebinding/SapServiceOperatorServiceBindingIoAccessor.java

```diff
@@ -20,30 +31,41 @@
 import java.util.stream.Collectors;
 import java.util.stream.Stream;
 
-import javax.annotation.Nonnull;
-import javax.annotation.Nullable;
-
-import org.json.JSONException;
-import org.json.JSONObject;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
-
-import com.sap.cloud.environment.servicebinding.api.DefaultServiceBinding;
-import com.sap.cloud.environment.servicebinding.api.ServiceBinding;
-import com.sap.cloud.environment.servicebinding.api.ServiceBindingAccessor;
-import com.sap.cloud.environment.servicebinding.api.exception.ServiceBindingAccessException;
-import com.sap.cloud.environment.servicebinding.metadata.BindingMetadata;
-import com.sap.cloud.environment.servicebinding.metadata.BindingMetadataFactory;
-import com.sap.cloud.environment.servicebinding.metadata.BindingProperty;
-
+/**
+ * A {@link ServiceBindingAccessor} that is able to load {@link ServiceBinding}s that conform to the
+ * <a href="https://servicebinding.io/spec/core/1.0.0/">servicebinding.io</a> specification with the
+ * <a href="https://blogs.sap.com/2022/07/12/the-new-way-to-consume-service-bindings-on-kyma-runtime/">SAP metadata
+ * extension</a> from the file system. <br>
+ * The file system structure is assumed to look as follows:
+ *
+ * <pre>
+ *     ${SERVICE-BINDING-ROOT}
+ *     ├-- {SERVICE-BINDING-NAME#1}
+ *     |   ├-- .metadata
+ *     |   ├-- {PROPERTY#1}
+ *     |   ├-- ...
+ *     |   └-- {PROPERTY#N}
+ *     └-- {SERVICE-BINDING-NAME#2}
+ *         ├-- .metadata
+ *         ├-- {PROPERTY#1}
+ *         ├-- ...
+ *         └-- {PROPERTY#N}
+ * </pre>
+ */
 public class SapServiceOperatorServiceBindingIoAccessor implements ServiceBindingAccessor
```

Comment:
why the `Io`? :D 
again, I'd consider cutting the "Sap" prefix

Reply:
Because this implementation refers to the [servicebinding.io](https://servicebinding.io/) specification. Just naming it `ServiceOperatorServiceBindingAccessor` would be too generic, right? (How would you explain the difference between the two accessor implementations?)

Reply:
Regarding the `Sap` prefix: Let's re-consider this for the next major version release. I've created [an issue](https://github.com/SAP/btp-environment-variable-access/issues/41) to not forget about this request.